relops_hardware_controller/settings.py

```diff
@@ -177,12 +183,12 @@ def CACHES(self):
     BUGZILLA_URL = values.URLValue(environ_prefix=None)
     BUGZILLA_API_KEY = values.SecretValue(environ_prefix=None)
 
-    XEN_URL = values.URLValue(environ_prefix=None)
-    XEN_USERNAME = values.Value(environ_prefix=None)
-    XEN_PASSWORD = values.SecretValue(environ_prefix=None)
+    XEN_URL = values.URLValue('', environ_prefix=None)
+    XEN_USERNAME = values.Value('', environ_prefix=None)
+    XEN_PASSWORD = values.Value('', environ_prefix=None)
 
-    ILO_USERNAME = values.Value(environ_prefix=None)
-    ILO_PASSWORD = values.SecretValue(environ_prefix=None)
```

Comment:
These were originally set as SecretValues to only let them be passed in as env vars.

Reply:
I'm planning to remove them and put all of the credentials into the json. I've removed these variables from the environment file in puppet. So I have them as not required here to avoid failure with the puppet patch's changes.